src/main/java/dev/Roach/GameLookup.java

```diff
@@ -0,0 +1,50 @@
+package dev.Roach;
+
+import dev.Roach.datamodel.game.Game;
+import dev.Roach.datamodel.game.GamePojo;
+import dev.Roach.datamodel.gameLookup.GameDealResponse;
+import dev.Roach.fetchers.GamesFetcher;
+
+import java.net.http.HttpClient;
+import java.util.ArrayList;
+
+public class GameLookup {
+    HttpClient client = HttpClient.newBuilder().build();
+    GamesFetcher gamesFetcher = new GamesFetcher(client);
+
+
+    public void giveTitleToGetListOFDealsWithStores(String gameTitle) {
```

Comment:
This method should be returning proper datamodel information rather than print it